docs/concepts/k8s/function-ingress.md

```diff
@@ -17,11 +17,20 @@ In addition to configuring a service, Nuclio creates a [Kubernetes ingress](http
 
 ## Setting up an ingress controller
 
-In this guide, you'll set up a [Træfik](https://docs.traefik.io/) controller, but any type of Kubernetes ingress controller should work. You can read [Træfik's excellent documentation](https://docs.traefik.io/user-guide/kubernetes/), but for the purposes of this guide you can simply run the following commands to set up the controller:
+In this guide, you'll set up a [Træfik](https://docs.traefik.io/) controller, but any type of Kubernetes ingress controller should work. You can read [Træfik's excellent documentation](https://docs.traefik.io/user-guides/crd-acme/), but for the purposes of this guide you can simply run the following 
+commands to set up the controller in one of two ways:
 
+- Directly using the resource YAMLs (note this installs v1.7, and was removed on newer versions):
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/containous/traefik/master/examples/k8s/traefik-rbac.yaml
-kubectl apply -f https://raw.githubusercontent.com/containous/traefik/master/examples/k8s/traefik-deployment.yaml
+kubectl apply -f https://raw.githubusercontent.com/containous/traefik/v1.7/examples/k8s/traefik-rbac.yaml
+kubectl apply -f https://raw.githubusercontent.com/containous/traefik/v1.7/examples/k8s/traefik-deployment.yaml
+```
+
+OR
+
+- Using Helm (provided helm is installed):
+```sh
+helm install stable/traefik --name traefik --namespace kube-system
```

Comment:
1. I would change "in one of two ways to "by using either of the following alternative methods" (although it's not critical).
2. Indent the code blocks by 2 characters to align with the content of the list items.
3. I suggest rephrasing the bullet to "Use `kubectl` to apply the YAML resource files directly, as demonstrated here for Traefik version 1.7:", although I wasn't sure what you meant by the "and was removed on newer versions" part of the note in your version, so maybe we need an additional edit to clarify.
4. You don't need the "OR" when you write "in one of two ways" (or my suggested variation) before the list.
5. I would change the second bullet to "Use [Helm](https://helm.sh/) (provided Helm is installed):".

Reply:
=> I've attached a revised version that implements all my comments:
[function-ingress.txt](https://github.com/nuclio/nuclio/files/4015440/function-ingress.txt)

Reply:
**Q:** Why do you use Traefik v1.7 in the first example but not in the second example?

Reply:
I've noticed that **docs/setup/aks/setup-aks-appinsights-grafana.md**  refers to "`helm`" and links to https://docs.helm.sh/. =>
1. https://docs.helm.sh/ redirects to https://helm.sh/docs/, so it's better to just link to the latter directly.
2. I would change the two instances of "`helm`" in the text to "Helm" (as the product name is spelt on their site), even though there's a `helm` CLI so the current version isn't "wrong".

Reply:
> "Q: Why do you use Traefik v1.7 in the first example but not in the second example?" - It seems that in later versions they simply removed the resource yamls from their repo so the `kubectl` method with a link to their github is not possible

> `Helm` vs `helm` -> like `kubectl` I'm referring to the CLI (`helm`) I think it's clear using lower case